README.md

```diff
@@ -1,129 +1,168 @@
+Currently, the industry standard frameworks are Hardhat and Truffle which both use JavaScript to test and deploy Solidity smart contracts. Foundry only uses Solidity for testing. This creates a much smoother developer experience as an engineer does not need to switch from one language to another.
 
-- [**Quick Start / Installation**](https://github.com/matter-labs/foundry-zksync#quick-start--installation)
-- [**v0.0 Feature Set**](https://github.com/matter-labs/foundry-zksync#feature-set)
-- [**Environment Variables**](https://github.com/matter-labs/foundry-zksync#environment-variables)
-- [**Blockchain Interaction**](https://github.com/matter-labs/foundry-zksync#blockchain-interaction)
-- [**Bridging Assets**](https://github.com/matter-labs/foundry-zksync#bridging-assets-l1--l2)
-- [**Compilation**](https://github.com/matter-labs/foundry-zksync#compilation)
-- [**Deployment**](https://github.com/matter-labs/foundry-zksync#deployment)
-- [**Contract Interaction**](https://github.com/matter-labs/foundry-zksync#contract-interaction)
-- [**Deploy and Interact with `SimpleFactory.sol`**](https://github.com/matter-labs/foundry-zksync#usage-example-simplefactorysol)
-- [**Account Abstraction Multisig example**](https://github.com/matter-labs/foundry-zksync#account-abstraction-multisig)
+### Supported features
 
+- Compile smart contracts with the [zksolc compiler](https://github.com/matter-labs/zksolc-bin).
+- Deploy smart contracts to zkSync Era testnet or local test node.
+- Bridge assets L1 <-> L2.
+- Call deployed contracts on zkSync Era testnet or local test node.
+- Send transactions to deployed contracts on zkSync Era testnet or local test node.
 
 ---
+
+## Contents
+
+- [Set up](https://github.com/matter-labs/foundry-zksync#set-up)
+- [Interact with blockchain](https://github.com/matter-labs/foundry-zksync#interact-with-blockchain)
+- [Compile](https://github.com/matter-labs/foundry-zksync#compile)
+- [Deploy](https://github.com/matter-labs/foundry-zksync#deploy)
+- [Bridge assets](https://github.com/matter-labs/foundry-zksync#bridge-assets-l1--l2)
+- [Interact with contract](https://github.com/matter-labs/foundry-zksync#interact-with-contract)
+- [Deploy and interact with `SimpleFactory.sol`](https://github.com/matter-labs/foundry-zksync#deploy-and-interact-with-simplefactorysol)
+- [Account abstraction multisig example](https://github.com/matter-labs/foundry-zksync#account-abstraction-multisig)
+
 ---
 
-### Quick Start / Installation
+## Set up
 
-Clone these two repos to the same directory:
+### Prerequisites
 
-- [**foundry-zksync**](https://github.com/matter-labs/foundry-zksync) - this is our application, we will be building this in the following steps
-- [**sample-fzksync-project**](https://github.com/sammyshakes/sample-fzksync-project) - this is the sample project that contains the smart contract to be compiled
+Check you have the zkSync Era [dev dependencies](https://github.com/matter-labs/zksync-era/blob/main/docs/setup-dev.md).
```

Comment:
this should not be needed (the stuff there is mostly for people developing the system itself)

Reply:
ok, but I guess I needed a bunch of pre-requisites ... can we list them please? i.e. Rust, etc .. this is really a placeholder for the actual prerequisites I'm unaware of ...

Reply:
in theory - you only need rust compiler